### DISCUSSION TITLE
Biome baseTemperature API

### DISCUSSION THREAD

**Discussion by LB45440078L** [PaperMC/Paper#10512] [Ideas]
### Is your feature request related to a problem?

It is currently impossible to get the base temperature of a vanilla Biome from its enum type.
These are hardcoded values used by the internals of the server that are however not exposed through the API.

As specified by the Minecraft Game Wiki:
![image](https://cdn.discordapp.com/attachments/555462289851940864/1221889868364386374/image.png?ex=6614389e&is=6601c39e&hm=9b4cd9780b1817288fea7a59d3ff80a443ac58460773371159c1a532a868619b&)

### Describe the solution you'd like.

A method into the API that is **not** bound and influenced by height parameters or data instances such as World or Block.

It would be something like double Biome#getBaseTemperature().

### Describe alternatives you've considered.

The current only alternative I have found for this issue is to use NMS or Reflections, but it's not a long term solution and inclusive for more more than one version

**Comment by OskarZyg** [comment]
Do you know where the base temperatures are? Your image of the wiki is dead and the list on https://minecraft.wiki/w/Biome#List_of_biome_climates is empty. If you have a class name, even better.

**Reply by Leguan16** [reply]
I can see it just fine. Are we talking about another list? ![image](https://github.com/PaperMC/Paper/assets/59799222/b0f8cd6c-aa85-4c12-b813-727675e1d1b7)

**Reply by OskarZyg** [reply]
weird, the content of that table doesn't appear for me.

by any chance, would you know where these values are hardcoded?

**Reply by Leguan16** [reply]
:)
![image](https://github.com/PaperMC/Paper/assets/59799222/550768cc-44db-4816-b407-1cd050ed021d)

**Comment by Leguan16** [comment]
Looking into the code I don't think it makes sense to address this functionality before we converted the Biome enum to a class (I think this is going to happen). Right now, you cannot directly add this method to Biome (without hardcoding every value to every biome) so you would have to add it to World. And this is ugly considering its just a temporary "fix" until we get BiomeType.
